tests/test_comparison.py

```diff
@@ -67,21 +68,25 @@ def circuit():
             qml.BasisState(np.array(basis_state), wires=0)
             one_qubit_block(wires=0)
             return qml.expval(qml.PauliZ(0))
+        
+        dev_l = lightning_dev_version(wires)
+        dev_d = lightning_dev_version(wires)
```

Comment:
Shouldn't the default qubit be called here?
I mean `dev_l` and `dev_d` seem to be the same.

Reply:
Ooops, copy-pasting issue. Thanks @AmintorDusko ! I'll fix for all of the following too